sdk/tests/batch/config/config_test.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import turing.batch.config
+import turing.generated.models
+
+
+@pytest.mark.parametrize(
```

Comment:
kind of a nitpick but thats one long decorator that is hard to see at first glance

Reply:
What's the suggestion for improvement? :)